deps/ledger-zxlib/dockerized_build.mk

```diff
@@ -1,328 +0,0 @@
-#*******************************************************************************
-#*   (c) 2019-2021 Zondax GmbH
-#*
-#*  Licensed under the Apache License, Version 2.0 (the "License");
-#*  you may not use this file except in compliance with the License.
-#*  You may obtain a copy of the License at
-#*
-#*      http://www.apache.org/licenses/LICENSE-2.0
-#*
-#*  Unless required by applicable law or agreed to in writing, software
-#*  distributed under the License is distributed on an "AS IS" BASIS,
-#*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#*  See the License for the specific language governing permissions and
-#*  limitations under the License.
-#********************************************************************************
-
-.PHONY: all deps build clean load delete check_python show_info_recovery_mode
-
-TESTS_ZEMU_DIR?=$(CURDIR)/tests_zemu
-EXAMPLE_VUE_DIR?=$(CURDIR)/example_vue
-TESTS_JS_PACKAGE?=
-TESTS_JS_DIR?=
-
-LEDGER_SRC=$(CURDIR)/app
-DOCKER_APP_SRC=/project
-DOCKER_APP_BIN=$(DOCKER_APP_SRC)/app/bin/app.elf
-
-DOCKER_BOLOS_SDKS=/project/deps/nanos-secure-sdk
-DOCKER_BOLOS_SDKX=/project/deps/nanox-secure-sdk
-DOCKER_BOLOS_SDKS2=/project/deps/nanosplus-secure-sdk
-
-# Note: This is not an SSH key, and being public represents no risk
-SCP_PUBKEY=049bc79d139c70c83a4b19e8922e5ee3e0080bb14a2e8b0752aa42cda90a1463f689b0fa68c1c0246845c2074787b649d0d8a6c0b97d4607065eee3057bdf16b83
-SCP_PRIVKEY=ff701d781f43ce106f72dc26a46b6a83e053b5d07bb3d4ceab79c91ca822a66b
-
-INTERACTIVE:=$(shell [ -t 0 ] && echo 1)
-USERID:=$(shell id -u)
-$(info USERID                : $(USERID))
-$(info TESTS_ZEMU_DIR        : $(TESTS_ZEMU_DIR))
-$(info EXAMPLE_VUE_DIR       : $(EXAMPLE_VUE_DIR))
-$(info TESTS_JS_DIR          : $(TESTS_JS_DIR))
-$(info TESTS_JS_PACKAGE      : $(TESTS_JS_PACKAGE))
-
-DOCKER_IMAGE_ZONDAX=zondax/builder-bolos:latest
-DOCKER_IMAGE_LEDGER=ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder:latest
-
-ifdef INTERACTIVE
-INTERACTIVE_SETTING:="-i"
-TTY_SETTING:="-t"
-else
-INTERACTIVE_SETTING:=
-TTY_SETTING:=
-endif
-
-UNAME_S := $(shell uname -s)
-ifeq ($(UNAME_S),Linux)
-	NPROC=$(shell nproc)
-endif
-ifeq ($(UNAME_S),Darwin)
-	NPROC=$(shell sysctl -n hw.physicalcpu)
-endif
-
-define run_docker
-	docker run $(TTY_SETTING) $(INTERACTIVE_SETTING) --rm \
-	-e SCP_PRIVKEY=$(SCP_PRIVKEY) \
-	-e BOLOS_SDK=$(1) \
-	-e BOLOS_ENV=/opt/bolos \
-	-u $(USERID) \
-	-v $(shell pwd):/project \
-	-e SUPPORT_SR25519=$(SUPPORT_SR25519) \
-	-e SUBSTRATE_PARSER_FULL=$(SUBSTRATE_PARSER_FULL) \
-	-e DISABLE_PREVIOUS=$(DISABLE_PREVIOUS) \
```

Comment:
We need these lines to be able to compile current/previous version.
They should be added in zxlib since we are using a submodule